compiler/nimeval.nim

```diff
@@ -93,8 +96,8 @@ proc findNimStdLib*(): string =
 proc findNimStdLibCompileTime*(): string =
   ## Same as ``findNimStdLib`` but uses source files used at compile time,
   ## and asserts on error.
-  const sourcePath = currentSourcePath()
-  result = sourcePath.parentDir.parentDir / "lib"
+  const exe = getCurrentCompilerExe()
+  result = exe.parentDir.parentDir / "lib"
```

Comment:
This should be ``result = exe.splitFile.dir.parentDir / "lib"``

Reply:
I know there are semantic differences bw parentDir and splitFile.dir (and IMO parentDir always has the more intuitive semantics; in fact splitFile still has some bugs right now), but why do you prefer splitFile.dir here? 

(parentDir does less work too but I'm not concerned with that here)

Reply:
``exe`` is a "full" filename and it has a dir component of which we select the parent dir. By your logic a filename's parentdir is the same as the directory it's in, that's confusing.

Reply:
I've made your suggestion change to unblock this PR but discuss this separately later